### ISSUE TITLE
[Accessibility] Double-tapping Edit Shipping/Coupons within the OrderForm in Create-mode does not open the edit screen.

### ISSUE THREAD

**Issue by joshheald** [woocommerce/woocommerce-ios#11359]
### Known issue

When creating an order, if you add a coupon or shipping line, then try to edit those before saving the order, you can't do that with voiceover.

It looks like it's to do with the `handleSwipeBackGesture`, from the HostingController the VO double-tap gesture on _just these rows_ is being propagated to `UIViewController.gestureRecognizer(_:shouldBeRequiredToFailBy:)` in `UINavigationController+Woo`, and compared against the swipe back gesture, which triggers the action sheet to confirm navigation away from the view.

We could add some workaround, but anything I can come up with right now has worse side-effects than the existing issue.

_Originally posted by @joshheald in https://github.com/woocommerce/woocommerce-ios/issues/11358#issuecomment-1841189540_

**Comment by joshheald** [comment]
Automated triage note: this was tested and commented by Codex using device/simulator automation. Tested on iPad Pro 11-inch (M4) simulator, iOS 18.6, WooCommerce iOS 24.9.0.4 on trunk df01963e5080f79ae419981ce6dc9630d291b484. I could open the New Order form, but I could not exercise this exact bug in the assigned automation lane because it depends on VoiceOver focus and VoiceOver double-tap activation of added Shipping/Coupon edit rows. Coordinate double-tap without VoiceOver is not equivalent to the reported gesture path, so I am marking this as not testable here rather than not reproduced.

**Comment by dangermattic** [comment]
Thanks for reporting! 👍

**Comment by joshheald** [comment]
This is no longer reproducible
